python/docs/source/community/index.md

```diff
@@ -0,0 +1,38 @@
+---
+html_theme.sidebar_secondary.remove: true
+---
+
+# Community
+
+Welcome to the Kaskada community!
+
+The Kaskada project was open-sourced in 2023. As a growing community, we're here to explore new ways to interact with event-based data at scale and have some fun along the way. We build in the open and use the channels below for communication.
+
+```{gallery-grid}
+:grid-columns: 1 2 2 3
+
+- header: "{fab}`github;pst-color-primary` GitHub Discussions"
+  content: "Ask questions and discuss Kaskada on GitHub."
+  link: https://github.com/kaskada-ai/kaskada/discussions
```

Comment:
Lost the `target=_blank` and `ref=noopener noreferrer` on these. We may want to see if we can have quarto add at least the `ref` part on all external links. The `target=_blank` is less useful -- people should right click if they want a new tab. Would actually be pretty annoying if all of our links popped up new tabs...

Reply:
https://github.com/quarto-dev/quarto-cli/discussions/7137